Jenkinsfile.merge

```diff
@@ -0,0 +1,51 @@
+pipeline {
+    agent {
+        label 'testintegration'
+    }
+
+    parameters {
+        choice(
+            name: 'STATUS',
+            choices: ['success-only', 'no-error', 'none'],
+            description: 'PR check status'
+        )
+        string(
+            name: 'MERGE_OPTIONS',
+            defaultValue: '-vvv --no-ask --reset --comment',
+            description: 'scc merge options'
+        )
+    }
+
+    environment {
+        // These optional environment variables can be set in
+        // Manage Jenkins → Configure System → Global properties → Environment variables
+
+        // MERGE_PUSH_BRANCH the branch that merges should be pushed to, default "merge"
+        // MERGE_GIT_USER the GitHub user that the repo branches should be pushed to
+        //   default is the owner of the CI node's GitHub token
+
+        BASE_REPO = 'omero-build.git'
+    }
+
+    stages {
+        stage('Merge') {
+            steps {
+                // build is in .gitignore so we can use it as a temp dir
+                sh 'mkdir build'
+                sh 'cd build && curl -sfL https://github.com/ome/build-infra/archive/master.tar.gz | tar -zxf -'
+                sh 'virtualenv build/venv --system-site-packages && build/venv/bin/pip install scc'
+                sh """
+                    . build/venv/bin/activate
+                    if [ "${env.MERGE_PUSH_BRANCH}" != "null" ]; then
```

Comment:
I couldn't find a way to check for a non-existent environment variable. Setting the sheel to `bash` and using the `${VAR:-}` syntax doesn't work, testing for `"null"` is the only reliable method I've found so far.